core/date_api.php

```diff
@@ -280,3 +280,62 @@ function print_date_selection_set( $p_name, $p_format, $p_date = 0, $p_default_d
 	}
 }
 
+/**
+ * Convert php date time format string to moment.js compatible format
+ * moment.js is a popular date time handling JS library. It is used by datetime picker library
+ * see https://stackoverflow.com/questions/30186611/php-dateformat-to-moment-js-format
+ * Moment.js date time format: https://momentjs.com/docs/#/parsing/string-format/
+ * Php date time format: https://www.php.net/manual/en/function.date.php
+ * @param string $p_php_format     	Datetime php format
+ * @return string 					Date time format string understood by moment.js based components
+ * @access public
+ */
+function convert_php_to_momentjs_datetime_format( string $p_php_format ) {
```

Comment:
In terms of naming consistency within date api, maybe the function should be renamed `date_convert_format_from_php_to_momentjs` or something similar

Reply:
We do still support PHP 5.5.x.
Type declararions / type hints for `string` type are not supported before PHP 7.0
https://www.php.net/manual/en/functions.arguments.php